bin/tblextr.py

```diff
@@ -239,13 +239,17 @@ def fill_ext_db(table_name, db, indir, trace_name, api_pid):
           continue
 
         if cid == 2:
+          if not pid in range_stack: range_stack[pid] = {}
```

Comment:
AS we discussed, the issue seem to be related to roctracer generated traces. We agreed that you will fix roctracer to generate the right traces. Also eventually move tblextr.py in roctracer repo instead of rocprofiler.

Reply:
I checked in new changes to tblextr.py to issue a warning when  pid not in range_stack or tid not in pid_stack. This change, along with the change to move hsa_init() earlier in the process should provide complete solution to the problem.